pom.xml

```diff
@@ -121,6 +121,14 @@
 			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
 		</repository>
 		<repository>
+			<id>jitpack.io</id>
```

Comment:
so that you can use jsonwikipedia in another pom/sbt project?
They act as some sort of mvn repo in which you can specify a url to a github repo and it will build it and give maven the jar of it.